src/bind.c

```diff
@@ -304,7 +304,7 @@ r_obj* as_df_row_impl(r_obj* x,
   // Remove names first as they are promoted to data frame column names
   x = KEEP(vec_set_names(x, r_null));
 
-  x = KEEP(vec_chop(x, r_null));
+  x = KEEP(vec_chop_unsafe(x, r_null, r_null));
```

Comment:
The version of `vec_chop()` we use internally has been renamed to `vec_chop_unsafe()` to make it clear that it doesn't perform `vec_as_location()` or `vec_as_chop_sizes()` checks on the inputs.

Similar to `vec_slice_unsafe()`!